code/game/objects/items/weapons/weaponry.dm

```diff
@@ -185,6 +185,7 @@
 	throwforce = 12
 	attack_verb = list("beat", "smacked")
 	w_class = WEIGHT_CLASS_HUGE
+	var/nextthrow_time = 0
```

Comment:
Minor thing but can you make this full snake_case? so something like `next_throw_time`. Reads just a bit better